requirements.txt

```diff
@@ -1,2 +1,2 @@
-git+https://github.com/PennyLaneAI/pennylane.git
+pennylane>=0.9
```

Comment:
We can actually change the requirement here back because no breaking code was added to the plugin (a `try` `except` structure was used).